src/org/rust/lang/structure/RustStructDeclFieldTreeElement.kt

```diff
@@ -0,0 +1,17 @@
+package org.rust.lang.structure
+
+import com.intellij.ide.structureView.StructureViewTreeElement
+import com.intellij.ide.structureView.impl.common.PsiTreeElementBase
+import com.intellij.psi.NavigatablePsiElement
+import org.rust.lang.core.psi.RustStructDeclField
+
+class RustStructDeclFieldTreeElement(element: RustStructDeclField) : PsiTreeElementBase<RustStructDeclField>(element) {
+
+    override fun getPresentableText(): String? {
```

Comment:
If you like you could use the expression body syntax here and elsewhere:

```Kotlin
override fun getPresentableText() = 
    "${element?.identifier?.text}: ${element?.typeSum?.text}"
```

There is an intention (`alt + Enter`) for switching between expression body and braced body.